### ISSUE TITLE
SyntaxError during execution due to breaking changes in std 0.210.0

### ISSUE THREAD

**Issue by SamPlacette** [antoniovizuete/djira#1]
This seemed to happen with the release of std 0.210.0 which ended support for several deprecated APIs.

Error output:
```
Warning Implicitly using latest version (0.210.0) for https://deno.land/std/encoding/base64.ts
Download https://deno.land/std@0.210.0/encoding/base64.ts
error: Uncaught SyntaxError: The requested module 'https://deno.land/std/encoding/base64.ts' does not provide an export named 'encode'
import { encode } from 'https://deno.land/std/encoding/base64.ts';
```

**Comment by SamPlacette** [comment]
See https://github.com/dfoobar/djira/pull/2

**Comment by SamPlacette** [comment]
Closed with #2
